packages/oruga/src/scss/components/_inputitems.scss

```diff
@@ -55,6 +55,9 @@ $inputitems-margin-icon-to-text: .1875em !default;
     &__input {
         border: none;
         box-shadow: none;
+        height: auto;
+        padding-top: 0.1em;
```

Comment:
@rubjo good catch! could you please use a scss variable for 0.1em value? Using a scss variable allows users to keep that value consistent across `padding-*` and `$inputitems-item-margin`.

Reply:
This is altered slightly now: instead of hardcoding a 0.1em padding-top, set vertical padding to 0 and use the same margin variable for the input as for the tag items. (Apart from left margin, set to 0 to avoid input breaking container border, same could be achieved with setting input background to transparent instead, I guess).

Reply:
Great @rubjo ! Acting on items looks more natural and intuitive.